source/2020-11-06-the-ember-times-issue-167.md

```diff
@@ -9,6 +9,7 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 New Release and APIs for ember-a11y-testing 🎉,
 New Octane ready release of the a11y focused ember-select-light 🎉,
+2 deprecation RFCs merged,
```

Comment:
I didn't have time to find an emoji. Feel free to add one.

Reply:
will do, thank you!